src/core/geometry/qgswkbtypes.h

```diff
@@ -114,9 +117,11 @@ class CORE_EXPORT QgsWkbTypes
       CurvePolygonZM = 3010,
       MultiCurveZM = 3011,
       MultiSurfaceZM = 3012,
+      TriangleZM = 3017,
       Point25D = 0x80000001,
       LineString25D,
       Polygon25D,
+      Triangle25D,
```

Comment:
Adding Triangle25D here will break the values of later enumeration items. xxx25D are for legacy stuff. No need for that for newer geometry types